datasets/vggface2.py

```diff
@@ -99,8 +99,11 @@ def __extract_gt(self):
         """
         Extracts the ground truth from the dataset
         """
-        detector = face_detection.build_detector("RetinaNetResNet50", confidence_threshold=.5,
-                                                 nms_iou_threshold=.4)
+        if torch.cuda.is_available():
+            detector = RetinaFace(gpu_id=torch.cuda.current_device(), network="resnet50")
+        else:
+            detector = RetinaFace(gpu_id=-1, network="resnet50")
```

Comment:
We need to modify this logic slightly. Currently, it's possible to specify "no GPU" even if one is available.
args.device in train.py holds the result of that logic.